cluster/node-pools/worker-karpenter/provisioners.yaml

```diff
@@ -0,0 +1,125 @@
+{{- if eq .NodePool.Profile "worker-karpenter" }}
+---
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: "{{.NodePool.Name}}"
+spec:
+  {{- if index .NodePool.ConfigItems "scaling_priority"}}
+  weight: {{.NodePool.ConfigItems.scaling_priority}}
+  {{- end}}
+  consolidation:
+    {{- if index .NodePool.ConfigItems "consolidation" }}
+    enabled: {{ or .NodePool.ConfigItems.consolidation true }}
```

Comment:
Do we need the `or` or can we use the plain value of `.NodePool.ConfigItems.consolidation` here?